isaric/parsers/isaric-rapid.json

```diff
@@ -148,11 +148,6 @@
         "3": null
       }
     },
-    "pregnancy_whether_breastfed": {
```

Comment:
Was this removed because it was duplicated in the parser? or why was it removed?